### PR TITLE
Rework ChatAdapterConnector to let multiple JAICP-integrated bots in single application

### PR DESCRIPTION
**Pull request by Denire**
Creating singleton ChatAdapterConnector was a bad design which led to inability maintain multiple JAICP-integrated bots in single application.

This resolves the issue, enables multiple CA connectors, therefore enables multiple JAICP-integrated bots.

### CODE REVIEW

File: channels/facebook/src/main/kotlin/com/justai/jaicf/channel/facebook/FacebookChannel.kt [hunk 1]
```diff
@@ -20,12 +22,13 @@ class FacebookChannel private constructor(
...
     private lateinit var messenger: Messenger
+    private var liveChatProvider: JaicpLiveChatProvider? = null
 
     constructor(botApi: BotApi, config: FacebookPageConfig) : this(botApi) {
         messenger = Messenger.create(config.pageAccessToken, config.appSecret, config.verifyToken)
     }
 
-    private constructor(botApi: BotApi, baseUrl: String) : this(botApi) {
+    private constructor(botApi: BotApi, baseUrl: String, liveChatProvider: JaicpLiveChatProvider) : this(botApi) {
         messenger = Messenger.create("", "", "", baseUrl)
```
Comment (veptechno) [inline channels/facebook/src/main/kotlin/com/justai/jaicf/channel/facebook/FacebookChannel.kt:32]: I guess here we need to assign liveChatProvider to the field

File: channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/channels/TelephonyChannel.kt [hunk 2]
```diff
@@ -45,14 +46,16 @@ class TelephonyChannel(
 
     companion object : JaicpNativeChannelFactory {
         override val channelType = "resterisk"
-        override fun create(botApi: BotApi) = TelephonyChannel(botApi)
+        override fun create(botApi: BotApi, liveChatProvider: JaicpLiveChatProvider): JaicpNativeBotChannel =
+            TelephonyChannel(botApi)
```
Comment (veptechno) [inline channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/channels/TelephonyChannel.kt:50]: Why isn't liveChatProvider used?

**Review by veptechno — CHANGES_REQUESTED** [review]
Please remove mapdb

File: channels/facebook/src/main/kotlin/com/justai/jaicf/channel/facebook/FacebookChannel.kt (hunk 1 above)
Comment (Denire) [reply channels/facebook/src/main/kotlin/com/justai/jaicf/channel/facebook/FacebookChannel.kt:32]: Thanks! Fixed

File: channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/channels/TelephonyChannel.kt (hunk 2 above)
Comment (Denire) [reply channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/channels/TelephonyChannel.kt:50]: Telephony channel does not support Live Chats

**Review by veptechno — APPROVED** [review]

**Review by morfeusys — APPROVED** [review]

**Pull request merged**